cmd/storer2elastic/dataprocessor/errors.go

```diff
@@ -34,3 +34,18 @@ var ErrNoMetachainDatabase = errors.New("no metachain database - cannot index")
 
 // ErrDatabaseInfoNotFound signals that a database information hasn't been found
 var ErrDatabaseInfoNotFound = errors.New("database info not found")
+
```

Comment:
Error on L27 is unused

Also in elastic/errors.go the first 3 errors are unused

Reply:
removed